testplan/common/entity/base.py

```diff
@@ -920,7 +922,10 @@ class ResourceConfig(EntityConfig):
     @classmethod
     def get_options(cls):
         """Resource specific config options."""
-        return {ConfigOption("async_start", default=True): bool}
+        return {
+            ConfigOption("async_start", default=True): bool,
+            ConfigOption("autostart", default=True): bool,
```

Comment:
Can we call it auto_start please? Having `autostart` right next to `async_start` looks a bit random to my eyes.